lib/ExpensiMark.js

```diff
@@ -153,6 +139,45 @@ export default class ExpensiMark {
                     return `${g1}<a href="${href}" target="_blank" rel="noreferrer noopener">${g2}</a>${g1}`;
                 },
             },
+            {
+                /**
+                 * Use \b in this case because it will match on words, letters,
+                 * and _: https://www.rexegg.com/regex-boundaries.html#wordboundary
+                 * The !_blank is to prevent the `target="_blank">` section of the
+                 * link replacement from being captured Additionally, something like
+                 * `\b\_([^<>]*?)\_\b` doesn't work because it won't replace
+                 * `_https://www.test.com_`
+                 * Use [\s\S]* instead of .* to match newline
+                 */
+                name: 'italic',
+                regex: /(\b_+|\b)(?!_blank")_((?![\s_])[\s\S]*?[^\s_])_(?![^\W_])(?![^<]*(<\/pre>|<\/code>|<\/a>|<\/mention-user>|_blank))/g,
+
+                // We want to add extraLeadingUnderscores back before the <em> tag unless textWithinUnderscores starts with valid email
+                replacement: (match, extraLeadingUnderscores, textWithinUnderscores) => {
+                    if (textWithinUnderscores.includes('<pre>') || this.containsNonPairTag(textWithinUnderscores)) {
+                        return match;
+                    }
+                    if (String(textWithinUnderscores).match(`^${CONST.REG_EXP.MARKDOWN_EMAIL}`)) {
+                        return `<em>${extraLeadingUnderscores}${textWithinUnderscores}</em>`;
+                    }
+                    return `${extraLeadingUnderscores}<em>${textWithinUnderscores}</em>`;
+                },
+            },
```

Comment:
We used to have the quote rule run before the italic rule. Changing that order caused an invalid markdown rendering for cases like:

```
_Msg then a
> quote_
```

Coming from https://github.com/Expensify/App/issues/26941